sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_generated/aio/__init__.py

```diff
@@ -6,5 +6,5 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from ._azure_digital_twins_api import AzureDigitalTwinsAPI
+from ._azure_digital_twins_api_async import AzureDigitalTwinsAPI
```

Comment:
This looks like a breaking change to me - as in other files, "_async" seems to be appended to many package names. Do we know why this is happening?